src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/SaveClaimWithCoreCaseDataStoreTest.java

```diff
@@ -28,8 +28,7 @@
         "feature_toggles.ccd_async_enabled=false",
         "feature_toggles.ccd_enabled=true"
     }
-)
-public class SaveClaimWithCoreCaseDataStoreTest extends BaseSaveTest {
+) public class SaveClaimWithCoreCaseDataStoreTest extends BaseSaveTest {
```

Comment:
something not right here?

Reply:
@dharmendrak next thing we need to incorporate in checkstyle. But revoking your approval for this is quite cruel :-)